assets/js/page-enhancements.js

```diff
@@ -82,7 +82,9 @@
 })($, document)
 ;(autoDownloadCoversheetPdf = ($, document) => {
   $(document).ready(() => {
-    $('#print-coversheets-content #downloadPdf').each((idx, element) => element.click())
+    setTimeout(() => {
```

Comment:
Delaying the auto-download on page render by 1 second is better as it allows the page to fully render and gives cypress a fighting chance of being able to assert the page properly. And 1 second doesnt impact the UX at all